batch/deployment.yaml

```diff
@@ -185,6 +170,10 @@ spec:
          secret:
            optional: false
            secretName: ssl-config-batch-driver
+       - name: ssh-public-key
+         secret:
+           secretName: batch-worker-ssh-public-key
```

Comment:
AFAICT, this does not exist in gcp k8s.

Reply:
That's why it's marked as optional. It does rely on the good faith that gcp won't try to use it. In this case it would fail by reading an empty mount (though the mount should still exist).